testdata/project-v3/api/v1/webhook_suite_test.go

```diff
@@ -81,17 +81,9 @@ var _ = BeforeSuite(func() {
 	err = AddToScheme(scheme)
 	Expect(err).NotTo(HaveOccurred())
 
-	err = admissionv1beta1.AddToScheme(scheme)
+	err = admissionv1.AddToScheme(scheme)
 	Expect(err).NotTo(HaveOccurred())
 
-	err = admissionv1beta1.AddToScheme(scheme)
-	Expect(err).NotTo(HaveOccurred())
-
-	err = admissionv1beta1.AddToScheme(scheme)
-	Expect(err).NotTo(HaveOccurred())
-
-	//+kubebuilder:scaffold:scheme
```

Comment:
We are removing `//+kubebuilder:scaffold:scheme` the tool will not able to inject the code here.
Could you please let us know why it is required?

Reply:
I think we don't need marker here because no adding scheme is needed to be injected after first scakffolding. If it is needed for future, I will restore this.

Reply:
It is a breaking change and cannot be addressed.
It would make the feature stop working for those who already scaffold the projects using go/v3 and want to create an api. So, could you please revet this one and ensure that we will still be using a marker with the changes proposed? 

For further information check the plugin versioning policy: https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/plugins/plugins-versioning.md

Reply:
/hold

Reply:
I restored the marker.